crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -796,6 +816,47 @@ impl AuthorityPerEpochStore {
         self.finish_consensus_transaction_process_with_batch(write_batch, key, consensus_index)
     }
 
+    /// Called when a ceritified checkpoint is inserted into the checkpoint store.
+    /// This function records the validators' participation in the certified checkpoint by
+    /// incrementing the counters stored in the epoch tables. Tallying score metrics are
+    /// also updated at this point to send the most up-to-date scores.
+    pub fn record_certified_checkpoint_signatures(
+        &self,
+        certified_checkpoint: &CertifiedCheckpointSummary,
+    ) -> Result<(), TypedStoreError> {
+        // TODO: what to do when the committee is not valid and an error is returned?
```

Comment:
i think you can do an unwrap or expect here - this error should be caught and handled much earlier.